ltx_video/pipelines/pipeline_ltx_video.py

```diff
@@ -887,6 +893,13 @@ def __call__(
             negative_prompt_attention_mask=negative_prompt_attention_mask,
             clean_caption=clean_caption,
         )
+
+        if load_needed_only:
+            self.text_encoder = self.text_encoder.cpu()
+
+        if load_needed_only:
```

Comment:
why not combine these into 
```

        if load_needed_only:
            self.text_encoder = self.text_encoder.cpu()
            self.transformer = self.transformer.cuda()
```

Reply:
of course, results in the same thing, i just did it this way bc i wanted to mentally separate the end of the text encoding step and the start of the next step a bit more